mailer.rst

```diff
@@ -272,6 +272,15 @@ Other Options
 
         $dsn = 'smtps://smtp.example.com?ping_threshold=200'
 
+``max_per_second``
+    The number of messages to send per second (0 to disable)::
```

Comment:
About this --> `0 to disable` What does it mean?

* Disable delivery: no messages will be sent
* Disable this option: the max number of messages is no longer limited

Thanks!

Reply:
2 is the right answer. explanation improved